src/vs/workbench/contrib/webviewPanel/browser/webviewEditorInput.ts

```diff
@@ -23,7 +23,7 @@ export class WebviewInput extends EditorInput {
 	}
 
 	public override get capabilities(): EditorInputCapabilities {
-		return EditorInputCapabilities.Readonly | EditorInputCapabilities.Singleton;
+		return EditorInputCapabilities.Readonly | EditorInputCapabilities.Singleton | EditorInputCapabilities.CanDropIntoEditor;
```

Comment:
Isn't this editor readonly?

Reply:
For webviews, editor/resource is readonly but the UI of webviews may still allow dropping files. In the markdown preview for example, I'd eventually like to allow you to drop an image into the preview to insert it in the editor 

Since we can't know what the UI of webviews will be like, I think we need to keep drop into enabled UI enabled